source/stylesheets/modules/_header.scss

```diff
@@ -91,6 +91,14 @@ $active-nav-color: #1d8feb;
 
 .header__brand {
   font-size: 30px;
+  
+  // Give brand focus outline on mobile
+  &:after {
```

Comment:
Unfortunately the `display: block;` ruins everything (it breaks the right floating menu button).